gen/services.tmpl

```diff
@@ -0,0 +1,65 @@
+{{- define "Services" }}
+{{- if eq .Driver "mysql" }}
+    services:
+      # Spin up a mysql:8 container to be used as the dev-database for analysis.
+      {{ .Driver }}:
+        image: mysql:8
+        env:
+          MYSQL_DATABASE: dev
+          MYSQL_ROOT_PASSWORD: pass
+        ports:
+          - 3306:3306
+        options: >-
+          --health-cmd "mysqladmin ping -ppass"
+          --health-interval 10s
+          --health-start-period 10s
+          --health-timeout 5s
+          --health-retries 10
+    {{- else if eq .Driver "postgres" }}
+    services:
+      # Spin up a postgres:15 container to be used as the dev-database for analysis.
+      {{ .Driver }}:
+        image: postgres:15
+        env:
+          POSTGRES_DB: dev
+          POSTGRES_PASSWORD: pass
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-start-period 10s
+          --health-timeout 5s
+          --health-retries 5
+    {{- else if eq .Driver "mariadb" }}
+    services:
+      # Spin up a mariadb:11 container to be used as the dev-database for analysis.
+      {{ .Driver }}:
+        image: mariadb:11
+        env:
+          MYSQL_DATABASE: dev
+          MYSQL_ROOT_PASSWORD: pass
+        ports:
+          - 3306:3306
+        options: >-
+          --health-cmd "healthcheck.sh --su-mysql --connect --innodb_initialized"
+          --health-interval 10s
+          --health-start-period 10s
+          --health-timeout 5s
+          --health-retries 10
+{{- end }}
+{{- end }}
+
+{{- define "UseServices" }}
+{{- $Driver := index . 0 }}
+{{- $Host := index . 1 }}
+{{- if eq $Driver "mysql" -}}
+          dev-url: 'mysql://root:pass@{{- $Host }}:3306/dev'
+{{- else if eq $Driver "postgres" -}}
+          dev-url: 'postgres://postgres:pass@{{- $Host }}:5432/dev?sslmode=disable'
```

Comment:
The default postgres database should have search_path=public (because other driver default to schema scope).